src/components/NavBar.js

```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const NavBar = () => <img src="../images/planet.png" alt="logo" />;
```

Comment:
Kindly fix the `src` attr for the `img` element, because it's not working

Reply:
Hi @Abdessittir 
Thanks for your review 

I know about it but I didn't add it because they want an empty nav but no problem I'll add it now 🙂